Rakefile

```diff
@@ -180,6 +180,7 @@ namespace :docs do
         end
 
         initialize_method.tags(:example).each do |tag|
+          padding = 34 # accounts for p-3 and border
```

Comment:
Not blocking but maybe we're getting to the point where could / should pull this markup and config into a erb template (or view component!)

Reply:
we have to revisit this whole pipeline, the Rakefile is getting unmaintainable